### ISSUE TITLE
New-AWSPowerShellLambda Warns missing AWSPowerShell.NetCore when using AWS.Tools

### ISSUE THREAD

**Issue by lantrix** [aws/aws-lambda-dotnet#546]
As per the recent AWS Tools for Powershell Version 4 GA Notice (aws/aws-tools-for-powershell#67) it is noted that:

> Starting with version 4.0 of AWS Tools for PowerShell, we suggest all users use AWS.Tools as their preferred option

Which I'm currently using:

```powershell
/Users/lantrix> Get-Module -Name AWS.Tools* -ListAvailable
    Directory: /Users/lantrix/.local/share/powershell/Modules

ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Binary     4.0.0.0    AWS.Tools.CloudFormation            Core,Desk {Get-CFNAccountLimit, Get-CFNChangeSet, Get-CFNChangeSetList, Get-CFNDetectedStackResourceDrift…}
Binary     4.0.0.0    AWS.Tools.Common                    Core,Desk {Clear-AWSHistory, Set-AWSHistoryConfiguration, Initialize-AWSDefaultConfiguration, Clear-AWSDefaultConfiguration…}
Binary     4.0.0.0    AWS.Tools.EC2                       Core,Desk {Add-EC2CapacityReservation, Add-EC2ClassicLinkVpc, Add-EC2InternetGateway, Add-EC2NetworkInterface…}
Binary     4.0.0.0    AWS.Tools.ElasticLoadBalancingV2    Core,Desk {Add-ELB2ListenerCertificate, Add-ELB2Tag, Edit-ELB2Listener, Edit-ELB2LoadBalancerAttribute…}
Binary     4.0.0.0    AWS.Tools.IdentityManagement        Core,Desk {Add-IAMClientIDToOpenIDConnectProvider, Add-IAMRoleTag, Add-IAMRoleToInstanceProfile, Add-IAMUserTag…}
Script     1.0.0.0    AWS.Tools.Installer                 Core,Desk {Uninstall-AWSToolsModule, Install-AWSToolsModule, Update-AWSToolsModule}
Binary     4.0.0.0    AWS.Tools.Lambda                    Core,Desk {Add-LMLayerVersionPermission, Add-LMPermission, Add-LMResourceTag, Get-LMAccountSetting…}
Binary     4.0.0.0    AWS.Tools.Organizations             Core,Desk {Add-ORGPolicy, Add-ORGResourceTag, Confirm-ORGHandshake, Deny-ORGHandshake…}
Binary     4.0.0.0    AWS.Tools.S3                        Core,Desk {Add-S3PublicAccessBlock, Copy-S3Object, Get-S3ACL, Get-S3Bucket…}
Binary     4.0.0.0    AWS.Tools.SecurityToken             Core,Desk {Convert-STSAuthorizationMessage, Get-STSAccessKeyInfo, Get-STSCallerIdentity, Get-STSFederationToken…}
```

As such when I create a Lambda from a template it will complain, being a _warning only_:

```powershell
/Users/lantrix> New-AWSPowerShellLambda -Template Basic -ScriptName TestLambda -Directory testlambda
WARNING: This script requires the AWSPowerShell.NetCore module which is not installed locally.
WARNING: To use the AWS CmdLets execute "Install-Module AWSPowerShell.NetCore" and then update the #Requires statement to the version installed.
If you are not going to use the AWS CmdLets then remove the #Requires statement from the script.
Created new AWS Lambda PowerShell script TestLambda.ps1 from template Basic at /Users/lantrix/testlambda
```

It's not a blocker, but it will become more relevant as people switch to AWS.Tools.

I looked at the code and decided against a PR, because a change to the `AWSLambdaPSCore.psm1` module would mean:

* A change to use AWS.Tools GA 4.0.0.0 would be dependency Major version bump
* I'm not sure how your team would want to handle dependencies currently listed for a given template, e.g.:
  * the possibility of `AWS.Tools` _or_ `AWSPowerShell.NetCore` in the blueprints (for example [`basic.ps1.txt`](https://github.com/aws/aws-lambda-dotnet/blob/master/PowerShell/Module/Templates/Blueprints/Basic/basic.ps1.txt#L12))

**Comment by normj** [comment]
Yes, I'm working on getting the module updated for the AWS.Tools release. I had hoped to get that done before re:Invent but looks like that isn't going to happen and I will finish up the work after re:Invent.

**Comment by ashishdhingra** [comment]
Hi @lantrix,

Good afternoon.

I was going through the issue backlog and came across this issue. Could you please confirm if this is fixed in the latest AWS Tools for PowerShell. When I execute the following command from PowerShell command line:

```PowerShell
New-AWSPowerShellLambda -Template Basic -ScriptName TestLambda -Directory testlambda
```

it generates the following output with no warning:
```
Configuring script to use installed version 4.1.2.0 of (@{ ModuleName = 'AWS.Tools.Common'; ModuleVersion = '4.0.5.0' }.Name)
Created new AWS Lambda PowerShell script TestLambda.ps1 from template Basic at /Users/ashdhin/dev/repros/testlambda 
```

Thanks,
Ashish

**Comment by lantrix** [comment]
> Hi @lantrix,
> 
> Good afternoon.
> 
> I was going through the issue backlog and came across this issue. Could you please confirm if this is fixed in the latest AWS Tools for PowerShell. When I execute the following command from PowerShell command line:

Hi yes I can confirm this is fixed in version `2.0.0.0` of `AWSLambdaPSCore`
